### DISCUSSION TITLE
Campaign module missing filters

### DISCUSSION THREAD

**Discussion by Marco31Bike** [YetiForceCompany/YetiForceCRM#16808] [How to]
Hi there!

I have installed two versions of 6.4 on the same server with perfect configuration. Both installed YF in Campaigns module don't display pick-list Select From Relation. Is it a part which needs to be configured somewhere?

![image](https://user-images.githubusercontent.com/121147294/212500449-411a56b5-9ae8-47b4-9f3f-8fe0d1764624.png)

**Comment by rskrzypczak** [comment]
In the current version, there is no longer a button to associate records from the filter. Bindings can only be made from the "Select record" button.
However, a list of filters from a given module can be added in the _Software configuration/Standard modules/Modules – Relations_ panel, although it only applies to narrowing down related records.
![image](https://user-images.githubusercontent.com/10221999/212616610-bbbb2f73-ab86-4456-8927-e865e8c292c7.png)

**Reply by s2b-git** [reply]
The disadvantage is that dialog where we selecting records doesn't allow to select ALL records, and when we move across the screen trying to select multiple lists, the previous selected is unmarked. The same happening on all list views of modules. This is invisible to most people here seems like but I assuming it as a sort of design bug, as such selection behavior is working well in all similar systems I know to date :-)

Best

**Comment by TonyM15** [comment]
I can see that gitdeveloper still have this preconfigured filters. Maybe misunderstanding?
![image](https://user-images.githubusercontent.com/35586696/213026991-103409be-42d6-41e9-92e3-b489328aa4be.png)

**Reply by rskrzypczak** [reply]
No.
Formerly, this picklist with filters was used to create new relationships.
![image](https://user-images.githubusercontent.com/10221999/213752544-0405893b-6ad6-4842-a11e-6e26d09bd7d6.png)

Currently, the picklist with filters is used only to narrow down the entries of records that are already related.
So the way they are used is completely different.
.. and as I mentioned before, by default this picklist with filters is no longer there, you need to add it in the configuration panel.

**Comment by Marco31Bike** [comment]
Hi, thank you for clarification. I checked this functionality, and it seems not working: both in my installed app and in git developer. GIF attached.
![filters](https://user-images.githubusercontent.com/121147294/213866177-79909e3d-c690-47bf-a1ca-4ee64f05c945.gif)

**Reply by rskrzypczak** [reply]
I don't think we fully understood each other.
There is no error here.
The functionality of creating relationships based on a filter (Load form filter button) is simply no longer there, it has been removed.

**Comment by Marco31Bike** [comment]
Hey, this is becoming complicated here. 
I will throw sample as follows: I have 7k accounts to which I would like to send a newsletter.
So, I created a Campaign for this certain purpose, and now I would like to insert 7k records (accounts) to related list. 
Can you please help me to understand how to do it without manually attaching records screen-by-screen using button Select Record?
As it was mentioned before, the popup doesn't allow for mass add all records from filter there. 7k records is a 140 times returning to Select Record button.

Help appreciated

**Reply by rskrzypczak** [reply]
You can try to send e-mails directly from the list of records in the Leads module.

**Comment by garryatbrighton** [comment]
[rskrzypczak](https://github.com/rskrzypczak) that is hardcore answer :-) 
Why just don't delete Campaigns module and rename project from YetiforceCRM to Yetiforce?
Is there a reason why Campaigns module functionality has been downgraded?
Just to remind everybody: CRM = Customers Relationship Management.

Don't take me wrong, I am trying to help....

**Reply by rskrzypczak** [reply]
@garryatbrighton this is just a suggestion to workaround the problem, because in the current version, the previously described functionality is simply not available.
Ultimately, this will be solved by the possibility of selecting all records from the modal window of the "Select record" button
